src/index.html

```diff
@@ -21,7 +21,360 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          alt="moyo logo"
+          class="header__logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav">
+          <li>
+            <a
+              href="#apple"
+              class="nav__link nav__link--is--active is-active"
+            >
+              apple
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#samsung"
+              class="nav__link"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#smartphones"
+              class="nav__link"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#laptops"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#gadgets"
+              class="nav__link"
+            >
+              gadgets
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#tablets"
+              class="nav__link"
+            >
+              tablets
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#photo"
+              class="nav__link"
+            >
+              photo
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#video"
+              class="nav__link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <article
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__image"></div>
```

Comment:
The div with the class card__image is empty. If this is an image, you should use the img tag instead and provide appropriate alt text for accessibility. If it's a container for an image, consider using CSS to add the image as a background.